simple_scheduler.gemspec

```diff
@@ -18,14 +20,17 @@ Gem::Specification.new do |s|
   DESCRIPTION
   s.license = "MIT"
 
+  s.required_ruby_version = ">= 2.7.0"
+
   s.files = Dir["{lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5.0"
   s.add_dependency "sidekiq", ">= 5.0"
   s.add_development_dependency "appraisal"
   s.add_development_dependency "rainbow"
   s.add_development_dependency "rspec-rails"
-  s.add_development_dependency "rubocop", "~> 0.66.0"
+  s.add_development_dependency "rubocop", "~> 1.28"
   s.add_development_dependency "simplecov", "< 0.18" # https://github.com/codeclimate/test-reporter/issues/413
   s.add_development_dependency "simplecov-rcov"
+  s.metadata["rubygems_mfa_required"] = "true"
```

Comment:
Move this up closer to that required_ruby_version just so it's not down by all the dependencies that we read together.